Makefile

```diff
@@ -1,5 +1,5 @@
 REGISTRY?=hub.docker.com
-REPOSITORY?=anmolbabu/supervisor
+REPOSITORY?=devanggaur7/supervisor
```

Comment:
Undo this change I think you have this for the purpose of verification but have a look at https://github.com/fabric8-hdd/supervisor/pull/38 we are moving away from the private repos.

Reply:
Should I revert back to "anmolbabu/supervisor" ? or to the new centos registry ?

Reply:
Revert back to old state the new changes will be done by #38